bridges/zap/version.go

```diff
@@ -0,0 +1,6 @@
+package zap // import "go.opentelemetry.io/contrib/bridges/zap"
+
+// Version is the current release version of sloghandler in use.
+func Version() string {
+	return "0.0.1-alpha"
+}
```

Comment:
Similar to what was suggested for the [slog bridge](https://github.com/open-telemetry/opentelemetry-go-contrib/blob/851b2fd97fa45bb502cc094b6e3e84464a140fc3/bridges/otelslog/version.go#L7), can you make this into an unexported const.